src/persist-client/src/impl/machine.rs

```diff
@@ -283,6 +283,7 @@ where
             // if the state change itself is _idempotent_, then we're free to
```

Comment:
do you want to add `tracing::instrument` on the `apply_unbatched_cmd` function as well?

Reply:
initially leaning no in the interest of keeping things clean. I've iterated on various permutations of what is/isn't instrumented while looking at the open-loop benchmark and it turns out to be easy to end up with so much that it's actually harder to see whats going on. e.g. for this one, in practice all the apply_unbatched_cmd callers are fairly thin wrappers around it

that said, i'm planning on immediately revisiting all this after getting a sense for what things looks like in real prod clusters+honeycomb on things people other than me have typed